packages/frontend-2/components/workspaces/Promo/Card.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <div class="flex flex-col border border-outline-3 rounded-lg pt-6 px-8 pb-10 h-full">
```

Comment:
Wonder if we can reuse the QuickStart card from the dashboard here if we add a icon slot, I think they are basically the same? Maybe we just need to rename it

Reply:
There are a few differences here. The padding is different, there is no CTA and the icon as you say. This is also likely to change as the content/design is not finalised. 

I also don't really like using components from other folders too much. At some point when either needs updated it causes more problems, because you don't expect QuickStartCard to be used elsewhere.

Reply:
I agree with the second point, however having multiple cards with is just a card with a border also isn't ideal, having icons/buttons optional isn't too bad, if we turn this into a more generic dashboard card (or whatever context) it doesn't have to cause problems :) We can leave it for now, but if we get more of these lets find a way to align them them (both design and FE)

Reply:
Created CommonCard and used it here